eclipse-collections/src/main/java/org/eclipse/collections/impl/factory/Sets.java

```diff
@@ -71,9 +71,9 @@
  * FixedSize Examples:
  *
  * <pre>
- * FixedSizeList&lt;String&gt; emptySet = Sets.fixedSize.empty();
- * FixedSizeList&lt;String&gt; setWith = Sets.fixedSize.with("a", "b", "c");
- * FixedSizeList&lt;String&gt; setOf = Sets.fixedSize.of("a", "b", "c");
+ * FixedSizeSet&lt;String&gt; emptySet = Sets.fixedSize.empty();
+ * FixedSizeSet&lt;String&gt; setWith = Sets.fixedSize.with("a", "b", "c");
+ * FixedSizeSet&lt;String&gt; setOf = Sets.fixedSize.of("a", "b", "c");
```

Comment:
Found this documentation mistake along the way.

Reply:
There are a couple of unrelated fixes in here. I don't mind, but if this doesn't get merged, they'll get lost.